src/test/ui/coherence/coherence-negative-outlives-lifetimes.rs

```diff
@@ -1,10 +1,15 @@
+// revisions: stock with_negative_coherence
```

Comment:
I just made this into a separate revision instead of a different test since it kinda makes sense for this to also be tracked here